.github/ISSUE_TEMPLATE.md

```diff
@@ -48,13 +58,39 @@ Need Support?
 <!-- Image tag (i.e. 1.0.0) -->
 
 #### Docker
-<!-- Run `docker version` -->
+
+```
+<Run `docker version` and paste here>
```

Comment:
For existing instructions for iotedged version ("Run `iotedge version`") are in a comment tag rather than a code block. Should we do the same here, for consistency?

Reply:
`docker version` is about 20 lines of output, so it should use the code block. `iotedge version` is just a one-liner, so I didn't make it use one. But it's nice to be consistent I suppose.

Reply:
Ah, ok. That makes sense. It's ok to be different. :)

Reply:
Running `docker version` on windows can result into the version of the parallel installed docker and not the moby docker iotedge installs and uses. Or even a command not found on windows (if no docker is installed and the moby docker cli is not in the path). I suggest you put some more instructions how to run docker with the right path and right pipe on windows. (I have not used iotedge on Linux yet)

Reply:
If docker is not installed or in PATH then there are bigger problems, and this information isn't needed any way.

Yes, Windows user needs to set `-H 'npipe:////./pipe/iotedge_moby_engine'` if they're using Moby, but using Docker is supported as a dev scenario (via `-ContainerOs Linux`), so we can't suggest one or the other unilaterally. I'll add a parenthetical that the user needs `-H ...` if they're using Moby.

`iotedge check` already fetches the engine version to compare if it's Moby or not, so we should just make it also include that information in the JSON output's `additional_info`. I'll do that in my other open PR for `iotedge check`. But that won't go out any time soon so we still need to ask here in the issue template for now.

Reply:
Done.